_src/web/getting-started/hello-world.md

```diff
@@ -97,32 +120,34 @@ Notice we are referencing the datasource we created by its assigned `key`.
 ```json
 {
   "page": {
-    "key": "Homepage"
+    "name": "Homepage"
   },
-  "datasources": {
+  "datasources": [
     "greeting"
-  },
-  "routes": [
-    {
-      "path": "/"
-    }
-  ]
+  ],
+  "route": {
```

Comment:
The original `routes` property is actually correct for Web 1.7+. See http://docs.dadi.tech/web/concepts/routing/#page-routing